Kitodo/src/main/java/org/kitodo/production/services/command/KitodoScriptService.java

```diff
@@ -85,7 +85,7 @@ public static KitodoScriptService getInstance() {
     /**
      * Private constructor. Use {@link #getInstance()} to get the instance.
      */
-    private KitodoScriptService() {
+    protected KitodoScriptService() {
```

Comment:
Why are you changing the constructor from `private` to `protected`?

Reply:
This was done so that the `KitodoScriptProcessorIT` can create a derived class from this one, that does not perform any functionality, but changes a state in the test class, so that you can check that the code to be tested is working correctly.